DESCRIPTION

```diff
@@ -49,7 +49,8 @@ Suggests:
     tzdb (>= 0.1.1),
     waldo,
     withr,
-    xml2
+    xml2,
+    vctrs
```

Comment:
Nitpick: our current style is to always keep packages in DESCRIPTION in alphabetical order. So vctrs should be inserted in the appropriate place in the list.

Description of this style and why:
https://r-pkgs.org/dependencies.html#run-time-vs-develop-time-dependencies

Note also the advice there re: usethis functions that help keep DESCRIPTION in line with these conventions.

Reply:
This is now irrelevant in this PR, but this convention should be noted for future changes to dependencies.